.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please help potential reviewers to understand this pull request and speed up 
+the process by answering the following questions, where appropriate.
+
+# Why was this pull request created?
+
+# How is this achieved?
+
+# How to test this?
```

Comment:
Really? We require textual description of tests again instead of automated tests?